### ISSUE TITLE
kubelet failed to start: k8s version is v1.24.15 and ubuntu 16.04

### ISSUE THREAD

**Issue by fuxiaoting** [kubernetes/kubernetes#119416]
### What happened?

kubelet failed to start due to below error.

/lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.32' not found (required by kubelet)
/lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by kubelet)

### What did you expect to happen?

kubelet starts successfully

### How can we reproduce it (as minimally and precisely as possible)?

kubernetes version ：v1.24.15 
worker os:ubuntu 16.04

Try to create a cluster, kubelet failed to start.

### Kubernetes version

<details>

```console
$ kubectl version :v1.24.15
# paste output here
```
/lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.32' not found (required by kubelet)
/lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by kubelet)

</details>

### Cloud provider

<details>
TKGi
</details>

### OS version

<details>

```console
# On Linux:
$ cat /etc/os-release
# paste output here
$ uname -a
# paste output here
ubuntu 16.04

# On Windows:
C:\> wmic os get Caption, Version, BuildNumber, OSArchitecture
# paste output here
```

</details>

### Install tools

<details>

</details>

### Container runtime (CRI) and version (if applicable)

<details>

</details>

### Related plugins (CNI, CSI, ...) and versions (if applicable)

<details>

</details>

**Comment by fuxiaoting** [comment]
Our golang version is still v1.17 , is it due to our golang version is so lower? Thank you.

**Comment by mingregister** [comment]
try upgrade your ubuntu version.

**Comment by adisky** [comment]
closing this issue since Kubernetes 1.24 released when ubuntu 16.04 was end of support and currently both are unsupported upstream.
/close
